src/map2d/translator.py

```diff
@@ -22,25 +22,60 @@ def __del__(self):
         if self.visualize:
             self.vis.destroy_window()
     
-    def normalize_values(self,points):
+    def normalize_values(self, points):
         """
-        Normalizes the z-values of the given points to be within the range [-1, 1].
+        Normalizes the y-values of the given points to be within the range [-1, 1].
         Args:
             points (np.array): Points extracted from the point cloud.
         Returns:
             np.array: The array of normalized points.
         """
-        z_min = np.min(points[:, 2])
-        z_max = np.max(points[:, 2])
-        points[:, 2] = 2 * (points[:, 2] - z_min) / (z_max - z_min) - 1
+        y_min = np.min(points[:, 1])
+        y_max = np.max(points[:, 1])
+        points[:, 1] = 2 * (points[:, 1] - y_min) / (y_max - y_min) - 1
         return points
 
+    def cropped_map_to_2d_map(self, points, position=(0, 0)):
+        """
+        Converts a point map to global 2D map.
+        Args:
+            points (np.array): The point map.
+            position (tuple): Position (x, z).
+        Returns:
+            np.array: The resulting 2D map.
+        """
+        # Map shape
+        p_height, p_width = points.shape
+
+        # Cut map
+        if position[0] + p_width > self.res:
+            p_width = self.res - position[0]
+        if position[1] + p_height > self.res:
+            p_height = self.res - position[1]
+
+        overlay_map = points[:p_height, :p_width]
+
+        # Assign values to the 2D map
+        for i in range(p_height):
```

Comment:
for  nie są zbyt optymalnym rozwiązaniem bo robimy to element po elemencie 
```python
zeros = np.zeros((5000, 5000))
global_map = zeros
small_matrix = np.random.rand(1000, 1000)  

start_row = 3434
start_col = 2344

global_map[start_row:start_row + small_matrix.shape[0], start_col:start_col + small_matrix.shape[1]] += small_matrix

```
to jest lepsze podejście bo wszytko się dzieje w numpy i nie są alokowane dodatkowe macierze na czas przepisania